doc/setting_up_sidewalk_environment/setting_up_sdk.rst

```diff
@@ -61,16 +61,7 @@ Follow these steps to download the Sidewalk application for the nRF Connect SDK:
        |___ zephyr
        |___ ...
 
-#. Check out the branch that contains the west manifest file with the Sidewalk group filter:
-
-   .. code-block:: console
-
-      $ cd nrf
-      $ git checkout main
-      $ git pull
-      $ cd ..
-
-#. Set the Sidewalk application manifest and update.
+#. Check manifest file. This step is necessary if you worked with Sidewalk as application with manifest repository before.
 
    Check the current manifest path and make sure it points to :file:`west.yml` inside the :file:`nrf` directory:
```

Comment:
```suggestion
```

Can we rephrase the current line 73 - ``If it does not, set the manifest path by running the following command:`` ? Changed the main sentence a bit so the following needs to match it gramatically, plus felt it could use some rewording.